src/controller/CHIPDevice.cpp

```diff
@@ -330,12 +340,13 @@ CHIP_ERROR Device::LoadSecureSessionParameters(ResetTransport resetNeeded)
     return err;
 }
 
-bool Device::GetIpAddress(Inet::IPAddress & addr) const
+bool Device::GetAddress(Inet::IPAddress & addr, uint16_t & port) const
 {
     if (mState == ConnectionState::NotConnected)
```

Comment:
nit for other PRs: UDP is by definition connectionless ... getting the IP address based on some 'connected' state sounds odd. Maybe it needs renaming (does it mean have valid node id or having a key?) or we should reconsider the flow.

Reply:
It means that the device is connected at CHIP level. That is, that keys have been exchanged etc. I agree this may be a bit confusing. Maybe `commissioned` would be a better name.